Packages/com.unity.inputsystem/InputSystem/Devices/TrackedDevice.cs

```diff
@@ -7,17 +7,85 @@ namespace UnityEngine.InputSystem
     /// <summary>
     /// An input device that has its orientation and position in space tracked.
     /// </summary>
+    /// <remarks>
+    /// These values are typically read from input actions and fed into the
+    /// [Tracked Pose Driver](xref:input-system-tracked-input-devices#tracked-pose-driver)
+    /// component rather than being read directly from this class.
+    ///
+    /// Refer to the [Starter Assets](https://docs.unity3d.com/Packages/com.unity.xr.interaction.toolkit@latest/index.html?subfolder=/manual/samples-spatial-keyboard.html)
+    /// sample in the XR Interaction Toolkit package for a Demo Scene with an XR rig
+    /// hierarchy that uses these concepts.
+    /// </remarks>
     /// <seealso cref="UnityEngine.InputSystem.XR.XRController"/>
     /// <seealso cref="UnityEngine.InputSystem.XR.XRHMD"/>
+    /// <seealso href="https://docs.unity3d.com/Packages/com.unity.xr.openxr@latest/index.html?subfolder=/api/UnityEngine.XR.OpenXR.Input.Pose.html">UnityEngine.XR.OpenXR.Input.Pose</seealso>
     [InputControlLayout(displayName = "Tracked Device", isGenericTypeOfDevice = true)]
     public class TrackedDevice : InputDevice
     {
+        /// <summary>
+        /// Indicates which of the tracked pose components are valid by using an integer containing a
+        /// bitwise OR of the [Unity XR module enum values](https://docs.unity3d.com/ScriptReference/XR.InputTrackingState.html),
+        /// for example `InputTrackingState.Position | InputTrackingState.Rotation`.
+        /// </summary>
+        /// <remarks>
+        /// This property determines whether you can retrieve valid values from the
+        /// <see cref="devicePosition"/> and the <see cref="deviceRotation"/> properties:
+        /// - The Position bit must be set for the <see cref="devicePosition"/> property to have a valid Vector3 value.
+        /// - The Rotation bit must be set for the <see cref="deviceRotation"/> property to have a valid Quaternion.
+        /// </remarks>
         [InputControl(synthetic = true)]
         public IntegerControl trackingState { get; protected set; }
+
+        /// <summary>
+        /// Indicates whether the input device is actively tracked or not.
+        /// </summary>
+        /// <remarks>
+        /// This property can contain a simple bit (1 or 0), but for some Open XR devices, this might
```

Comment:
```suggestion
        /// This property can contain a simple bit (1 or 0), but for some OpenXR devices, this might
```
OpenXR doesn't have a space.